tests/frameworks/frameworkBar/BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+load("//rules:framework.bzl", "apple_framework_packaging")
+load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
+
+swift_library(
+    name = "frameworkBar",
```

Comment:
What do you mean by `frameworkBar`? What is being tested in this example?

Reply:
Can give some suggestion on the name? In this case what happened is there is a target to compile a swift library called `frameworkBar`  that depends `FrameworkFoo`, then it was packaged into a framework with a name `Unique_FrameworkBar ` which also depends on `FrameworkFoo`. Without the change introduced in this PR, it will result in redefinition of module `FrameworkBar` because it is not properly propagated through the swift info. Does this makes sense?